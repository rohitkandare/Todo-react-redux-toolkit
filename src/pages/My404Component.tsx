import './page.css'

function My404Component() {
  
  return (
    <div className='notFound'>
      <h1>404 Not Found </h1>
    </div>
  );
}

export default My404Component;
