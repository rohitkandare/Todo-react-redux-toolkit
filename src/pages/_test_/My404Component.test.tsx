import { mount, shallow } from "enzyme";
import React from "react";
import My404Component from "../My404Component";
import toJson from 'enzyme-to-json';

describe('AddTodoForm component', () => {
  it('renders correctly enzyme', () => {
    const wrapper = shallow(<My404Component />)
    expect(wrapper.find('h1').text()).toEqual('404 Not Found ');
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
