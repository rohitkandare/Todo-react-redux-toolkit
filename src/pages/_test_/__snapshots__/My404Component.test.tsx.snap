// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTodoForm component renders correctly enzyme 1`] = `
<div
  className="notFound"
>
  <h1>
    404 Not Found 
  </h1>
</div>
`;
